libraries/AP_HAL_ChibiOS/hwdef/Aerium-RadianH743/README.md

```diff
@@ -0,0 +1,123 @@
+  - 2 × CAN (only 1 exposed on the FPV carrier board)
+  - 2 × I2C (only 1 exposed on the FPV carrier board)
+  - 3 × SPI (internal)
+  - 1 × RC input
+  - 16 × PWM/servo outputs (only 8 exposed on the FPV carrier board)
+  - 1 × microSD card slot (for logging)
+  - 2 × ADC inputs (no exposed on FPV carrier board)
+  - 1 × onboard FFC connector for stabilized/dampened heated IMU
+- **Weight:** Approximately 20 g  
+- **Dimensions:**
+  - **Carrier board:** 46 × 38 mm (M4 mounting holes for M3 dampers, 30.5 × 30.5 mm spacing)  
+  - **Radian core module:** 32 × 32 mm (M2 mounting holes with integrated standoffs)
+
+<img src="https://aerium.bitbucket.io/documentation/assets/images/radian-dimensions-3eba0ff37b66f5fc1343267414ca2eb3.jpg#diagram" width="35%" alt="Radian Dimensions">
+
+---
+
+## Purchase
+
+- Available at the [Aerium online store](https://www.aerium.co.il/product-page/radian)
+
+---
+
+## Connectors and Pinouts
+
+<img src="https://aerium.bitbucket.io/documentation/assets/images/radian-pinout-top-50f03ed9e3b93c7910bc5317b7327ac3.jpg#diagram" width="35%" alt="Radian Top View">
+
+<img src="https://aerium.bitbucket.io/documentation/assets/images/radian-pinout-bottom-bd16e46157ac40b959d25613e66bdec4.jpg#diagram" width="35%" alt="Radian Bottom View">
+
+For detailed pin definitions and usage, refer to the [Radian documentation portal](https://aerium.bitbucket.io/documentation/Product%20Manuals/Radian/).
+
+---
+
+## UART Port Mapping
+
+| ArduPilot SERIALx | Connector | Function                              |
+|-------------------|-----------|----------------------------------------|
+| SERIAL1           | TELEM1    | Primary telemetry                      |
+| SERIAL2           | TELEM2    | Secondary telemetry or companion link |
+| SERIAL3           | GPS       | GNSS module                            |
+| SERIAL3           | GPS       | GNSS module                            |
+| SERIAL3           | GPS       | GNSS module                            |
+| SERIAL3           | GPS       | GNSS module                            |
+
+---
+
+## PWM / Servo Outputs
+
+- Provides 8 independent PWM or servo outputs  
+- Functions can be assigned using `SERVOx_FUNCTION` parameters in supported firmware  
+
+---
+
+## RC Input
+
+- Supports PPM-Sum, S.Bus, and iBUS protocols via the RC input port  
+- Additional receiver types can be connected via configurable serial ports  
+- For setup instructions, consult the [ArduPilot RC systems guide](https://ardupilot.org/copter/docs/common-rc-systems.html)
+
+---
+
+## Power & Battery Monitoring
+
+- Dual onboard voltage sensors for independent monitoring of both power inputs  
+- Supports external voltage and current monitoring through power connectors  
+- Default firmware configurations are pre-mapped to the correct pins  
+- Use onboard VSense jumpers to toggle between internal and external sensing for each input  
+- Voltage and current scalers can be adjusted for precise measurements  
+
+---
```

Comment:
needs an RSSI section, AIRSPEED section 
see https://ardupilot.org/copter/docs/common-aocoda-h743dual.html for an example